app/models/case_contact.rb

```diff
@@ -145,6 +145,12 @@ def active_or_expenses?
 
   scope :no_drafts, ->(checked) { (checked == 1) ? where(status: "active") : all }
 
+  scope :drafts_for_removal, -> {
+    where.not(status: "active")
+      .where("case_contacts.created_at < ?", 1.week.ago)
+      .or(where(draft_case_ids: []).where("case_contacts.created_at < ?", 1.day.ago))
```

Comment:
I'm not sure what the best names would be but can we turn `1.week.ago` into named constants as opposed to magic values?

Reply:
will do